src/sources/cfg.rs

```diff
@@ -0,0 +1,59 @@
+use crate::LintLevel;
+use serde::Deserialize;
+
+#[derive(Deserialize)]
+#[serde(rename_all = "kebab-case", deny_unknown_fields)]
+pub struct Config {
+    #[serde(default = "crate::lint_warn")]
+    pub unknown_registry: LintLevel,
+
+    #[serde(default = "crate::lint_warn")]
+    pub unknown_git: LintLevel,
+
+    //    #[serde(default = "crate::lint_warn")]
+    //    pub missing_git_revision: LintLevel,
+    #[serde(default = "default_allow_registry")]
+    pub allow_registry: Vec<String>,
+
+    #[serde(default)]
+    pub allow_git: Vec<String>,
```

Comment:
I wonder if this should be able to just do allowances by user/org as well (at least for github sources), eg, 
```toml
allow-git = ["https://github.com/bytecodealliance"]
```

Reply:
Then again, generally you don't have so many git sources that that's a big deal.

Reply:
Yes  I'm thinking so as well, was first thinking of it as URL wildcard matching (see "Add wildcard URL searches?" section). But could potentially be additional config option here also, though implementation would be the same. 

Though the concept of an org is a GitHub concept in their URL, not a git one in general. So doing filtering/pattern matching on the git repo URL may be the cleanest.

An easy option would be to just check that the URL matches the length of it, then one can have a shorter URL with just the org name in the allow list